cmd/nodecmd/create_aws.go

```diff
@@ -46,27 +46,27 @@ func getNewKeyPairName(ec2Svc *ec2.EC2) (string, error) {
 	}
 }
 
-func printNoCredentialsOutput() {
+func printNoCredentialsOutput(awsProfile string) {
 	ux.Logger.PrintToUser("No AWS credentials file found in ~/.aws/credentials")
```

Comment:
No AWS credentials found. Either set AWS environment variables ENV_VAR_NAMES_HERE or set config file ~/.aws/credentials.
Improve this printed msg to be clear there are two options (env vars or credential file, what to do with either of those, and where to found more info)

Reply:
Also be very clear that, in the file,  it is looking for the profile awsProfile. So as to close https://github.com/ava-labs/avalanche-cli/issues/986

Reply:
done